src/lib/reasoners/enum_rel.ml

```diff
@@ -195,6 +222,7 @@ let count_splits env la =
   in
   {env with size_splits = nb}
 
+(* Add the uninterpreted semantic value [r] to the environment. *)
```

Comment:
Same as above, it is clear from the name that we add `[r]` to the environment, but what does it mean? Apparently we initialize the corresponding `mx` entry with all the possible constructors for its type.